lib/api-client.js

```diff
@@ -5,6 +5,10 @@ var apiClient = new JSONAPIClient(config.host + '/api', {
   'Content-Type': 'application/json',
   'Accept': 'application/vnd.api+json; version=1',
 }, {
+  params: {
+    http_cache: true
```

Comment:
For testing, the same if i provide a URL param then it should pick it up via config and set this param to  `http_cache: true` so it activates the cache. 
Once we're happy with it we can have it on by default and use the query param to turn it off.